connectors/kafka/src/main/java/io/camunda/connector/kafka/inbound/KafkaExecutable.java

```diff
@@ -63,13 +65,23 @@ public class KafkaExecutable implements InboundConnectorExecutable<InboundConnec
   private final Function<Properties, Consumer<Object, Object>> consumerCreatorFunction;
   public KafkaConnectorConsumer kafkaConnectorConsumer;
 
+  private final RetryPolicy<Object> retryPolicy;
+
   public KafkaExecutable(
-      final Function<Properties, Consumer<Object, Object>> consumerCreatorFunction) {
+      final Function<Properties, Consumer<Object, Object>> consumerCreatorFunction,
+      final RetryPolicy<Object> retryConfig) {
     this.consumerCreatorFunction = consumerCreatorFunction;
+    this.retryPolicy = retryConfig;
   }
 
   public KafkaExecutable() {
-    this(KafkaConsumer::new);
+    this(
+        KafkaConsumer::new,
+        RetryPolicy.builder()
+            .handle(Exception.class)
+            .withDelay(Duration.ofSeconds(30))
+            .withMaxAttempts(-1)
```

Comment:
maybe having a constant here to express the meaning: INFINITE or similar (if this is what -1 means)

Reply:
Good point!